GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -670,6 +672,9 @@ private void InitializeComponent()
             // MainSplitContainer.Panel1
             // 
             this.MainSplitContainer.Panel1.Controls.Add(this.repoObjectsTree);
+            this.MainSplitContainer.Panel1.Padding = new System.Windows.Forms.Padding(1);
+            this.MainSplitContainer.Panel1.BackColor = Color.FromArgb(217, 217, 217).AdaptBackColor();
```

Comment:
We can't do this in the designer-generated file. This will be lost the next time this file is opened in the designer